oem/firstvoices/windows/src/inst/Makefile

```diff
@@ -73,15 +73,15 @@ clean:
 
 check:
     if not exist $(ROOT)\src\engine\inst\keymanengine.msm $(MAKE) check-engine
-    if not exist $(ROOT)\src\global\inst\data\unicodedata.mdb $(MAKE) check-unicodedata
+    if not exist $(KEYMAN_ROOT)\common\windows\bin\data\unicodedata.mdb $(MAKE) check-unicodedata
```

Comment:
```suggestion
    if not exist $(KEYMAN_ROOT)\common\windows\bin\data\UnicodeData.mdb $(MAKE) check-unicodedata
```

Reply:
I've gone with moving remaining 2 refs to lower case (it was inconsistent previously anyway!)